packages/graphql/src/translate/create-connect-or-create-and-params.ts

```diff
@@ -52,6 +52,8 @@ export function createConnectOrCreateAndParams({
     context: Context;
     withVars: string[];
 }): CypherBuilder.CypherResult {
+    const withVarsVariables = withVars.map((name) => new CypherBuilder.RawVariable(name));
```

Comment:
**question:** I remember us having a conversation about parameters vs variables and I thought we agreed on variables would always be "raw" (i.e. directly in the Cypher). Do we need the "Raw" prefix here?

Reply:
Here RawVariable refers to a variable that already has a name, maybe "NamedVariable" is a better name for this.

The difference is:
```
new CypherBuilder.Variable(); // Will get translated to a unique var0
new CypherBuilder.Variable("my-var"); // Will get translated to my-var
```

This class is purely for compatibility reasons.